internal/impl/timeplus/output.go

```diff
@@ -0,0 +1,216 @@
+package timeplus
+
+import (
+	"context"
+	"errors"
+	"sort"
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/redpanda-data/connect/v4/internal/impl/timeplus/http"
+)
+
+var outputConfigSpec *service.ConfigSpec
+
+func init() {
+	// TODO: add Version
+	outputConfigSpec = service.NewConfigSpec().
```

Comment:
What would be the `Version` here? Shall I just use the next minor version (`4.38.0`)? 🤔

Reply:
Don't worry about adding it, it's being deprecated soon as we're recording the versions separately going forwards.

Reply:
Gotcha, thanks!